### DISCUSSION TITLE
Why not AutoGluonTS?

### DISCUSSION THREAD

**Discussion by mreismendes** [autogluon/autogluon#822] [Ideas]
It could be amazing having an implementation of "auto" in GluonTS.

Is there any plan is this way? Something in this way: https://github.com/winedarksea/AutoTS

Thank you!

**Comment by Innixma** [comment]
Integration between GluonTS and AutoGluon is actively being worked on. @jwmueller Can you provide more details?

**Reply by hantingxie** [reply]
Sounds great! Is the work going to be based on HPO only or Ensembling?

**Comment by jwmueller** [comment]
For time-series there are 3 plans in progress: 

1) Add AutoGluon-TabularPrediction as a new model available in GluonTS:
https://github.com/awslabs/gluon-ts/pull/1105 

2) Add a new `Forecasting` prediction task in AutoGluon (that uses GluonTS internally):
https://github.com/awslabs/autogluon/pull/684 

3) Add stack-ensembling for time-series into AutoGluon's `Forecasting` task (work in progress still in the early stages).

@hantingxie @mreismendes 
If interested, please take a look at those PRs and leave review comments if you have suggestions.

We'd also love it if you're interested in contributing some time-series functionality!  If you prefer to discuss via email, you can reach out at <autogluon-dev@amazon.com>, or happy to discuss ideas in this thread! We could also provide some pointers if you'd like to hear some contribution ideas.
